lib/spack/spack/util/environment.py

```diff
@@ -874,9 +874,17 @@ def from_sourcing_file(
             ]
         )
 
+        # Implements a no op during forwards operation on respective os
```

Comment:
This comment should go below (to the "before" part), although as it is I think something more like:

> First look at the environment after doing nothing to establish baseline

Might be clearer.

Reply:
Done!